static/app/utils/analytics/tracingEventMap.tsx

```diff
@@ -55,7 +55,6 @@ export type TracingEventParameters = {
   'trace_explorer.search_success': {
     has_data: boolean;
     num_traces: number;
-    project_platforms: string[];
```

Comment:
@KevinL10 is the project platforms necessary for analytics? It's being delegated to a separate query so it's no longer to possible to get a full list anymore.